fairlearn/postprocessing/_postprocessing.py

```diff
@@ -58,15 +69,27 @@ def predict(self, X, *, sensitive_features):
 
     def _pmf_predict(self, X, *, sensitive_features):
         """ Probabilistic mass function
+
+        :param X: Feature matrix
+        :type X: numpy.ndarray or pandas.DataFrame
+        :param sensitive_features: Sensitive features to identify groups by, currently allows
+            only a single column
+        :type sensitive_features: Currently 1D array as numpy.ndarray, list, pandas.DataFrame,
+            or pandas.Series
+        :return: array of tuples with probabilities for predicting 0 or 1, respectively
```

Comment:
Wouldn't we have $p(0) = 1 - p(1)$? I know we're not really supporting `_pmf_predict` but that's not a very clear explanation of the return type.

Reply:
That's correct. What would you suggest to make it clearer?

Reply:
I'm not sure, since I don't know the actual return type? Isn't it an array of predictors with a label "chose this one with this probability?"

Reply:
It's an array of tuples where the first entry is the prob for 0 and the second one is the prob for 1

Reply:
OK. I suppose those have to sum to 1?

Reply:
Right. I can add that

Reply:
done